main/res/layout/search_activity.xml

```diff
@@ -167,8 +160,7 @@
                 android:id="@+id/trackable"
                 style="@style/textinput_embedded"
                 android:hint="@string/search_tb_hint"
-                android:imeOptions="actionGo"
-                android:inputType="textVisiblePassword" />
+                android:imeOptions="actionGo" />
```

Comment:
Here as well.

I know textVisiblePassword was used for it before, and it is definitely still necessary. Even better use "textCapCharacters|textVisiblePassword". that will disable auto suggestions and start the keyboard in Capital Letters mode which is consistent as typed letters are converted to capital anyway.

Reply:
Alternatively, you could use `textNoSuggestions` instead of `textVisiblePassword`. But at least one of both.

Reply:
The `textCapCharacters` did not work on my device.
So I assume that the IME problems are not fixed by the keyboard software creators.
Therefore lets go back to `textVisiblePassword` and stay by the old implementation.

Reply:
See eg. discussion here: https://stackoverflow.com/questions/3789905/how-can-i-turn-off-suggestions-in-edittext .